src/auth/routes/userRotes.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const bcrypt = require('bcrypt');
+const express = require('express');
+const router = express.Router();
+
+const Users = require('../models/userSchema');
+const basicAuth = require('../middleware/authentication');
+
+router.post('/signup', async (req, res) => {
+  try {
+    req.body.password = await bcrypt.hash(req.body.password, 10);
+    const user = new Users(req.body);
+    const record = await user.save(req.body);
+    res.status(200).json(record);
+  } catch (e) { res.status(403).send("Error Creating User"); }
```

Comment:
if you want you can change this line to `catch (e) { next(e.message) }` which can tell you a little bit more details on what error you are getting.